lighthouse-core/gather/driver/wait-for-condition.js

```diff
@@ -112,10 +113,10 @@ function waitForFcp(session, pauseAfterFcpMs, maxWaitForFcpMs) {
  * `networkQuietThresholdMs` ms and a method to cancel internal network listeners/timeout.
  * @param {LH.Gatherer.FRProtocolSession} session
  * @param {NetworkRecorder} networkMonitor
```

Comment:
any reason this isn't called `networkRecorder`?

Reply:
just an artifact of how it used to be in driver, though honestly `networkMonitor` does better express it's usage almost everywhere other than for devtoolsLog creation :)

Reply:
it always confused me b/c there is no network-monitor file. would be nice to rename one or the other